### DISCUSSION TITLE
I can't lock down my beta release, what am I doing wrong?

### DISCUSSION THREAD

**Discussion by tking16** [appium/appium#17367] [Q&A] [answered]
I have an install command in my Makefile which currently runs `npm install appium@2.0.0-beta.40`. 

However whenever this is run over a fresh install it still picks up the very latest updates....

**Comment by jlipps** [comment]
@tking16 can you be more specific about what's happening? What does "the very latest updates" mean?

**Reply by tking16** [reply]
@jlipps Appium at the time of writing is on version `Appium v2.0.0-beta.43`. Why is my framework grabbing this version if I specifically requested beta version 40 using `npm install appium@2.0.0-beta.40`?

**Reply by KazuCocoa** [reply, marked as answer]
You did not have a global installed appium and package.json except for current directly (it may has the beta.40), right?
